index.js

```diff
@@ -0,0 +1,177 @@
+const inquirer = require("inquirer");
+const db = require("./database/connection.js");
+require("console.table");
+
+async function start() {
+  mainQuestions();
+}
+
+async function mainQuestions() {
+  const res = await inquirer.prompt([
+    {
+      type: "list",
+      name: "Choices",
+      message: "Please choose one of the options or add:",
+      choices: [
+        "View all departments",
+        "View all roles",
+        "View all employees",
+        "Add a department",
+        "Add a role",
+        "Add an employee",
+        "Update an employee role",
+        "Exit",
+      ],
+    },
+  ]);
+  let choice = res.Choices;
```

Comment:
Please change this to const as the variable is not reassigned its value within the scope of this function